e2e/cypress/integration/integrations/builtin_commands/common_commands_spec.js

```diff
@@ -243,6 +256,28 @@ describe('I18456 Built-in slash commands: common', () => {
                 });
         });
     });
+
+    it('MM-T658 /invite - current channel', () => {
+        loginAndVisitDefaultChannel(user1, `${team1.name}/channels/${testChannel.name}`);
+
+        // # Post `/invite @username` where username is a user who is not in the current channel
+        cy.postMessage(`/invite @${userGroup[0].username}`);
+
+        // * User who added them sees system message "username added to the channel by you"
+        cy.uiWaitUntilMessagePostedIncludes(`@${userGroup[0].username} added to the channel by you`);
+
+        // * Cannot invite deactivated users to a channel
+        cy.postMessage(`/invite @${deactivatedUser.username}`);
+        cy.uiWaitUntilMessagePostedIncludes('We couldn\'t find the user. They may have been deactivated by the System Administrator.');
+
+        cy.apiLogin(userGroup[0]);
+        cy.visit(`${team1.name}/channels/town-square`);
+
+        // * Added user sees channel added to LHS, mention badge, system message "username added to the channel by username."
```

Comment:
I don't see the "mention badge". Could someone please point out if the badge supposed to be there?